Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormsCheckFurtherQuestionsExempted.cshtml

```diff
@@ -1,57 +1,58 @@
 @page "/organisation/{organisationId:guid}/forms/{formId:guid}/sections/{sectionId:guid}/check-further-questions-exempted"
 
 @model FormsCheckFurtherQuestionsExemptedModel
+@using CO.CDP.Localization
 
 @{
     var hasError = ((TagBuilder)Html.ValidationMessageFor(m => m.Confirm)).HasInnerHtml;
     ViewData["Title"] = Model.Heading;
 }
 
 @section BeforeContent {
-    <a href="/organisation/@Model.OrganisationId/supplier-information" class="govuk-back-link">Back</a>
+    <a href="/organisation/@Model.OrganisationId/supplier-information" class="govuk-back-link">@StaticTextResource.Global_Back</a>
```

Comment:
I see so even the "back" button text is now stored into the Resource file. >@StaticTextResource.Global_Back

Reply:
All of it! Every last word :-)